client/src/rpc_config.rs

```diff
@@ -11,12 +11,20 @@ pub struct RpcSignatureStatusConfig {
     pub search_transaction_history: bool,
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub enum RpcTransactionEncoding {
```

Comment:
I think I'd rather reuse `UiTransactionEncoding` here, and then just reject the additional encodings in code

Reply:
Ah!  There _is_ a `UiTransactionEncoding`. I was looking in all of the wrong places :sweat_smile:

Reply:
Updated in https://github.com/solana-labs/solana/pull/12683/commits/584d682b62f5f3efa7015147854fb6ee74c894c2